src/scripts/components/ResourceIntro.jsx

```diff
@@ -23,6 +23,26 @@ class ResourceIntro extends React.Component {
     }
   }
 
+  renderCheckbox = () => {
+    const { checkbox } = this.props;
+    const classes = cx('resource__intro__media', {
+      'resource__intro__media--hidden@xsmall': this.props.hideMediaXsmall,
+    });
+    const output = (
```

Comment:
Do we need this `output` variable rather than just returning the checkbox directly?